tradesman/model_creation/import_population.py

```diff
@@ -17,7 +17,16 @@ def import_population(project: Project, model_place: str, source: str, overwrite
         raise ValueError("Could not find a population file to import")
 
     df = population_raster(url, f"pop_{model_place}", project)
+    gdf = gpd.GeoDataFrame(df, geometry=gpd.points_from_xy(df.longitude, df.latitude), crs=4326)
+
+    model_area = gpd.read_postgis(
+        "SELECT ST_AsBinary(geometry) as geom FROM political_subdivisions WHERE level=-1", con=project.conn
+    )
+
+    select_pop = gdf.sjoin(model_area)
 
-    df.to_sql("raw_population", project.conn, if_exists="append", index=False)
+    select_pop[["longitude", "latitude", "population"]].to_sql(
```

Comment:
You can move the selection of columns to line 26 and avoid this line skipping because of length